platforms.c

```diff
@@ -136,6 +136,10 @@ static void pv_platform_set_status(struct pv_platform *p, plat_status_t status)
 		return;
 
 	p->status.current = status;
+	pv_log(DEBUG, "platform '%s' status is now %s", p->name,
```

Comment:
if this is the real status message that we are after, lets use INFO level at least.....